episodes/multi-dimensional-images.md

```diff
@@ -0,0 +1,449 @@
+
+- Explain how axes are handled in Napari (xyz, channels, time)
+- Open and navigate images with different dimensions in Napari
+- Explain what RGB images are and how they are handled in Napari
+- Split and merge channels in Napari
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Image dimensions / axes
+
+As we saw in the ['What is an image?' episode](./FIXME.md), image pixel values 
+are stored as arrays of numbers with certain dimensions and data type. So far we 
+have focused on grayscale 2D images that are represented by a 2D array:
+
+![](fig/array.png){alt="A diagram comparing the array of numbers and image 
+display for a simplified image of an arrow" width='80%'}
+
+Light microscopy data varies greatly though, and often has more dimensions 
+representing:
+
+- Time (t)
+- Channels (c)
+- Spatial axes (z / y / x)
+
+These images will be stored as arrays with a larger number of dimensions than 2. 
+Let's start with our familiar human mitosis image, and work up to some more 
+complex imaging data.
+
+## 2D
+
+Go to the top menu-bar of Napari and select:  
+`File > Open Sample > napari builtins > Human Mitosis`
+
+![](fig/human-mitosis-napari.png){alt="A screenshot of a 2D image of human cells 
+undergoing mitosis in Napari"}
+
+We can see this image only has two dimensions (or two 'axes' as they're also 
+known) due to the lack of sliders under the image, and by checking its shape in 
+the Napari console:
+
+```python
+image = viewer.layers["human_mitosis"].data
+print(image.shape)
+```
+
+```output
+# (y, x)
+(512, 512)
+```
+
+## 3D
+
+Let's remove the mitosis image and move to a 3D image:  
+`File > Open Sample > napari builtins > Brain (3D)`
+
+![](fig/brain-napari.png){alt="A screenshot of a head X-ray in Napari"}
+
+This image shows part of a human head X-ray CT dataset. We can see it has three 
+dimensions due to the slider at the base of the image, and the shape output:
+
+```python
+image = viewer.layers["brain"].data
+print(image.shape)
+```
+
+```output
+# (z, y, x)
+(10, 256, 256)
+```
+
+This 3D image can be thought of as a stack of 2D images, where the z axis points 
+along the stack. It is stored as a 3D array:
+
+![](fig/2d-3d-arrays.png){alt="A diagram comparing 2D and 
+3D image arrays" width='80%'}
+
+In Napari (and Python in general), dimensions are referred to by their index 
+e.g. here dimension 0 is the z axis, dimension 1 is the y axis and dimension 2 
+is the x axis. We can check this in Napari by looking at the number at the very 
+left of the slider. Here it's labelled '0', showing that it controls movement 
+along dimension 0 (i.e. the z axis).
+
+## Channels
+
+Next, let's look at a 3D image with channels. Remove the brain image and select:  
+`File > Open Sample > napari builtins > Cells (3D+2Ch)`  
+
+![](fig/cells-napari.png){alt="A screenshot of a flourescence microscopy image 
+of some cells in Napari"}
+
+Channels are used e.g. with fluorescence microscopy to store images collected 
```

Comment:
Could this be more general about channels. for example. 
"Image channels can be used to store data from multiple sources for the same pixel. Examples of this include images taken at different times (time series data) or taken with different wavelengths."

Reply:
Happy to add a more general description, but I don't think it's a good idea to mention time here. At least in microscopy, time and channels are usually treated separately as it gets confusing when you have e.g. a 3D time series which also has fluorescent channels (tczyx).